### ISSUE TITLE
Feature request: Make urlchecker work outside of packages?

### ISSUE THREAD

**Issue by njtierney** [r-lib/urlchecker#43]
Thanks for writing this, this has saved me a lot of time in package submission.

I was wondering if there might be scope to make this work outside of a package context? For example testing the links in a qmd or rmd file?

**Comment by gaborcsardi** [comment]
Maybe.

**Comment by Aariq** [comment]
Yeah, this would be nice.  I've got a quarto book with lots of links to university web pages (which are notorious for disappearing with no redirects) and it'd be cool to have a GitHub action that checked for broken links occasionally.

**Comment by Aariq** [comment]
Perhaps useful to others finding this issue: https://github.com/lycheeverse/lychee
